themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties

```diff
@@ -571,6 +571,10 @@ authorization-url=Authorization URL
 authorization-url.tooltip=The Authorization Url.
 token-url=Token URL
 token-url.tooltip=The Token URL.
+client-auth=Client Authentication
+client-auth.tooltip=The way client id and secret should be sent to the server cfr. https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication.
```

Comment:
The stuff was already added to this file in the other PR. I think that in your PR, you don't need to touch this file at all.